atd-vzd/migrations/default/1697145115428_add_ems_table/up.sql

```diff
@@ -0,0 +1,430 @@
+
+
+--
+-- Name: COLUMN ems__incidents.mvc_form_patient_injured_flag; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.mvc_form_patient_injured_flag IS 'Indicates whether the patient was injured ( 1 = yes, 0 = no)';
+
+
+--
+-- Name: COLUMN ems__incidents.mvc_form_position_in_vehicle; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.mvc_form_position_in_vehicle IS 'Where the patient was in the vehicle (front seat, second seat, etc)';
+
+
+--
+-- Name: COLUMN ems__incidents.mvc_form_safety_devices; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.mvc_form_safety_devices IS 'A list of any safety devices used, such as seatbelts or car seats';
+
+
+--
+-- Name: COLUMN ems__incidents.mvc_form_seat_row_number; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.mvc_form_seat_row_number IS '?';
+
+
+--
+-- Name: COLUMN ems__incidents.mvc_form_vehicle_type; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.mvc_form_vehicle_type IS 'The type of vehicle involved in the accident (automobile, motorcycle, etc)';
+
+
+--
+-- Name: COLUMN ems__incidents.mvc_form_weather; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.mvc_form_weather IS 'A general description of weather conditions at the time of the accident';
+
+
+--
+-- Name: COLUMN ems__incidents.pcr_additional_agencies; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.pcr_additional_agencies IS 'A comma delimitted list of agencies that responded to this incident in addtion to EMS';
+
+
+--
+-- Name: COLUMN ems__incidents.pcr_transport_priority; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.pcr_transport_priority IS 'Code 1 is lower priority transport without lights and sirens. Code 3 is a higher priority transport with lights and sirens';
+
+
+--
+-- Name: COLUMN ems__incidents.pcr_patient_acuity_initial; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.pcr_patient_acuity_initial IS 'Initial patient acuity determined by provider';
+
+
+--
+-- Name: COLUMN ems__incidents.pcr_patient_acuity_final; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.pcr_patient_acuity_final IS 'Final patient acuity determined by provider';
+
+
+--
+-- Name: COLUMN ems__incidents.apd_incident_numbers; Type: COMMENT; Schema: public; Owner: vze
+--
+
+COMMENT ON COLUMN public.ems__incidents.apd_incident_numbers IS 'A comma delimitted list of incident numbers for APD incidents that are linked to the EMS incident. This field can be used to determin if there is an associated APD incident.';
```

Comment:
this is such good metadata 😭

Reply:
agreed! i want to thank whoever wrote this 🙏